src/app/my-file/components/my-file/my-file.component.scss

```diff
@@ -0,0 +1,76 @@
+ .app-file-container{
+    padding-top: 10%;
+    padding-bottom: 20%;
+    width: 100%;
+    height:100%;
+  margin: auto;
+}
+
+.app-file-container h2{
+    font-size: 100% !important;
+}
```

Comment:
What is this used for?

Reply:
Nothing. Leftover from trying to adapt Heading Size in file component.